code/modules/surgery/organs/organ_external.dm

```diff
@@ -680,6 +686,17 @@ Note that amputating the affected organ does in fact remove the infection from t
 		tbrute = 3
 	return "[tbrute][tburn]"
 
+/obj/item/organ/external/proc/update_splints()
+	if(!(status & ORGAN_SPLINTED))
+		return
+	if(owner.step_count >= splinted_count + SPLINT_LIFE)
+		status &= ~ORGAN_SPLINTED //oh no, we actually need surgery now!
+		owner.visible_message("<span class=danger>[owner] screams in pain as their splint pops off their [src.name]!</span>","<span class=userdanger>You scream in pain as your splint pops off your [src.name]!")
```

Comment:
it seems weird that you'd scream out in pain if your leg/whatever's splint fell off if your limb was fine, but eh whatever pre-splinters can go eat a ham